src/include/common/rpp/handle.hpp

```diff
@@ -83,11 +84,12 @@ struct Handle : rppHandle
 {
     // Host handle related
     Handle();
-    Handle(size_t nBatchSize);
+    Handle(size_t nBatchSize, Rpp32u numThreads);
```

Comment:
probably should set the default value here too

Reply:
Done